tests/tests/fee_tests.cpp

```diff
@@ -263,7 +263,7 @@ BOOST_AUTO_TEST_CASE(asset_claim_pool_test)
         if( db.head_block_time() <= HARDFORK_CORE_188_TIME )
         {
            // can't claim pool before hardfork
-           GRAPHENE_REQUIRE_THROW( claim_pool( alice_id, bobcoin_id, _core(200), core_asset), fc::exception );
+           GRAPHENE_REQUIRE_THROW( claim_pool( alice_id, aliceusd.id, _core(200), core_asset), fc::exception );
```

Comment:
Perhaps I didn't make it clear enough. Should fund the fee pool with enough CORE before this test.

Reply:
Well, that test fails because we are befor the hardfork.
After going forward to the hardfork right below that code, the next test fails because
the fee pool is not funded but we are after the hard fork.
Right after that, the fee pool is funded and we do further tests.

Am I misunderstanding something?